functionalTest/test/specs/example.e2e.js

```diff
@@ -0,0 +1,81 @@
+const dashboard = require('../pageobjects/dashboard.page');
+const reusableObj = require('../utilities/reusableLibrary');
+
+
+describe('Covid-19 Dashboard', () => {
+    it('should have correct title', () => {
+        dashboard.open();
+        console.log("Url Launched....");
+        browser.pause(1000);
```

Comment:
It's a bit more advanced, but it would be good to not have sleeps/pauses/waits in the code. Ideally, we should be able to tell when the browser  had finished all asynchronous calls and rendering, and continue the testing then. This one is ok for now, but we should look at a way of safely removing it before all the pauses build up.

Reply:
Was using browser.pause for learning purposes. Removed all waits in the latest commit.